src/helpers.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+#include <unistd.h>
+#include "debug.h"
+
+void print_help()
+{
+    printf("Usage: simple-calc [OPTIONS]\n");
```

Comment:
This is fine for now but the simplecalc spec will want the input in the form operand1 operator operand2 (like ./simplecalc 2 + 2)

Reply:
Small nitpick but you can pass in argv[0] to this in case your executable name changes rather than hardcoding it in